tests/Aspire.Hosting.Tests/Qdrant/AddQdrantTests.cs

```diff
@@ -174,7 +174,7 @@ public async Task QdrantClientAppWithReferenceContainsConnectionStrings()
     public async Task VerifyManifest()
     {
         var appBuilder = DistributedApplication.CreateBuilder(new DistributedApplicationOptions() { Args = new string[] { "--publisher", "manifest" } } );
-        var qdrant = appBuilder.AddQdrant("qdrant");
+        var qdrant = appBuilder.AddQdrant("qdrant", httpPort:5503);
```

Comment:
Instead of modifying this test, we should add a new one. I can push a new test to this branch.